app/assets/javascripts/surveys.js

```diff
@@ -3,41 +3,55 @@
+  }
 
   var URL_SURVEY_TEMPLATE = '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
                             '  <div class="wrapper">' +
                                  cancelButton +
-                            '    <h1 id="feedback-heading">Tell us what you think of GOV.UK</h1>' +
-                            '    <p><a href="javascript:void()" id="take-survey" target="_blank" rel="noopener noreferrer">Take the 3 minute survey</a> This will open a short survey on another website</p>' +
+                                 surveyTitle +
+                            '    <p>' +
+                                 takeSurveyLink('Take the 3 minute survey', 'survey-primary-link') +
+                            '    This will open a short survey on another website</p>' +
                             '  </div>' +
                             '</section>',
     EMAIL_SURVEY_TEMPLATE = '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
                             '  <div class="wrapper">' +
                                  cancelButton +
                             '    <div class="inner-wrapper">' +
-                            '      <h1 id="feedback-heading">We\'d like to hear from you</h1>' +
+                                   surveyTitle +
                             '      <div id="email-survey-pre">' +
-                            '        <p><a href="#email-survey-form" id="email-survey-open" rel="noopener noreferrer" role="button" aria-expanded="false">Your feedback will help us improve this website</a></p>' +
+                            '        <a class="survey-primary-link" href="#email-survey-form" aria-labelledby="survey-title email-survey-open" id="email-survey-open" rel="noopener noreferrer" role="button" aria-expanded="false">' +
+                            '          Take a short survey to give us your feedback' +
+                            '        </a>' +
                             '      </div>' +
                             '      <form id="email-survey-form" action="/contact/govuk/email-survey-signup" method="post" class="js-hidden" aria-hidden="true">' +
                             '        <div id="feedback-prototype-form">' +
-                            '          <label for="survey-email-address">Tell us your email address and we\'ll send you a link to a quick feedback form.</label>' +
+                            '          <div id="survey-form-description" class="survey-form-description">We\'ll send you a link to a feedback form. It only takes 2 minutes to fill in.<br> Don\'t worry: we won\'t send you spam or share your email address with anyone.</div>' +
+                            '          <label class="survey-form-label" aria-describedby="survey-form-description" for="survey-email-address">' +
+                            '            Email Address' +
```

Comment:
This is consistent with how our forms work, and will allow users that use Zoom functionality to understand which form they're interacting with.

With the prototype if you zoomed in on the input field you wouldnt know what it's for, whereas in this iteration the label is nearby and easy to check.